tools/cli/admin.go

```diff
@@ -214,6 +214,48 @@ func newAdminShardManagementCommands() []cli.Command {
 	}
 }
 
+func newAdminMembershipCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:  "list_gossip",
+			Usage: "List cluster membership heartbeats",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagClusterMembershipRole,
+					Value: "all",
+					Usage: "Membership role filter: all (default), frontend, history, matching, worker",
+				},
+			},
+			Action: func(c *cli.Context) {
+				AdminListMembers(c)
+			},
+		},
+		{
+			Name:    "list_db",
```

Comment:
You may want to just push this to the other PR and join them together since much of this is a re-review

Reply:
sry removed the Draft flag too early. Updated the current PR so it doesn't show changes from the other